src/clr-angular/forms/combobox/all.spec.ts

```diff
@@ -19,7 +19,8 @@ import ComboboxOptionIntegrationSpecs from './combobox-with-option.integration.s
 import ComboboxOptionsMenuIntegrationSpecs from './combobox-with-options-menu.integration.spec';
 import ComboboxSpecs from './combobox.spec';
 
-describe('Combobox component', function() {
+// disable until combobox is merged
+xdescribe('Combobox component', function() {
```

Comment:
Because they refer the old (pre-Jeeyun, or early Jeeyun) code, which does not even surface anywhere. They _are_ getting changed in the new combobox implementation. I _could_ make these tests work for the old code (which, I remind, is not visible to no one, it's just some WIP), but I'll have to overwrite it anyway.
I am commenting them to skip this intermediate and unnecessary step. This will also make my life easier when merging later, because I'll have to only merge commented old code and my new changes. In the other case I'll have to be merging the fixed old code and my new code.

Reply:
Updated this one too to Jeremy's proposed comment format